packages/multimedia/ffmpeg/package.mk

```diff
@@ -89,6 +93,13 @@ pre_configure_target() {
 
 # ffmpeg fails running with GOLD support
   strip_gold
+
+  if [ "$KODIPLAYER_DRIVER" = "bcm2835-driver" ]; then
+    export CFLAGS="-I$SYSROOT_PREFIX/usr/include/interface/vcos/pthreads -I$SYSROOT_PREFIX/usr/include/interface/vmcs_host/linux -DRPI=1 $CFLAGS"
+    export FFMPEG_LIBS="-lbcm_host -lvcos -lvchiq_arm -lmmal -lmmal_core -lmmal_util -lvcsm"
+  else
+    export LDFLAGS="$LDFLAGS -fPIC"
```

Comment:
Actually I can't remember why it was done like that. I've just tested with `-fPIC` for RPi2 and it's working for HEVC and non-HEVC so I'll update the commit back to how it was with `-fPIC` for everything.